code/modules/vore/fluffstuff/custom_items_yw.dm

```diff
@@ -538,4 +538,35 @@
 	. = ..()
 	if (M == user)
 		M.resize(3)
-		M.drop_from_inventory(src)
\ No newline at end of file
+		M.drop_from_inventory(src)
+
+// //////////////////// //
+// Telescoping Unbrella //
+// //////////////////// //
+
+/obj/item/weapon/melee/umbrella/telescoping
+	name = "telescoping umbrella"
+	desc = "A compack telescoping umbrella,use with caution on windy days.."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "umbrella_closed"
+	addblends = "umbrella_closed_a"
+	slot_flags = SLOT_BELT
+	force = 2 //Its smaller
+	throwforce = 3
```

Comment:
Need to put the small `w_class` here as well, as when created it'll be normal-sized until open and closed.